src/index.html

```diff
@@ -20,8 +21,348 @@
+          <p class="card__amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product's photo"
+          class="card__photo"
+        />
+        <p class="card__describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product's photo"
+          class="card__photo"
+        />
+        <p class="card__describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="product's photo"
+          class="card__photo"
+        />
+        <p class="card__describe">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__text">Price:</p>
+          <p class="card__amount">$2,199</p>
+        </div>
+        <a
+          href="#"
```

Comment:
Consider replacing the '#' in the href attribute with a valid URL or JavaScript function to provide meaningful navigation or action when the 'Buy' button is clicked.